go/glide.yaml

```diff
@@ -0,0 +1,14 @@
+package: github.com/Originate/exocom/go
+import:
+- package: github.com/fatih/color
+  version: v1.1.0
+- package: golang.org/x/net
+  subpackages:
+  - websocket
+- package: gopkg.in/tylerb/graceful.v1
+  version: v1.2.13 
```

Comment:
`github.com/fatih/color` and `gopkg.in/tylerb/graceful.v1` are unused and should be removed.